phalcon/Cache.zep

```diff
@@ -103,6 +105,8 @@ class Cache implements CacheInterface
      */
     public function get(var key, var defaultValue = null) -> var
     {
+        let key = (string) key;
```

Comment:
It is possible to change method signature -> `public function get(string key)`

Reply:
Same as above